client/layout/index.jsx

```diff
@@ -131,6 +135,9 @@ Layout = React.createClass( {
 					<WelcomeMessage welcomeSite={ newestSite } />
 				</Welcome>
 				<TranslatorInvitation isVisible={ showInvitation } />
+				<PreferencesData>
+					<PollInvitation isVisible={ ! disablePollInvitation } section={ this.props.section } />
```

Comment:
Yeah, I feel the same way.  We're only planning on running this one for a couple of weeks, but I think the next one needs to be a reusable component.